internal/clients/team2/conversion.go

```diff
@@ -24,3 +26,28 @@ func magnitude(v ForceVector) float64 {
 func cosineSimilarity(v1, v2 ForceVector) float64 {
 	return dotProduct(v1, v2) / (magnitude(v1) * magnitude(v2))
 }
+
+// Normalizes the values of a map to be between 0 and 1.
+// Useful when calculating social capital from Networks, Reputation, and Institutions
+func normalizeMapValues(capitals map[uuid.UUID]float64) map[uuid.UUID]float64 {
+	if len(capitals) == 0 {
+		return capitals
+	}
+
+	min := 0.0
+	max := 0.0
+	for _, value := range capitals {
+		if value < min {
+			min = value
+		}
+		if value > max {
+			max = value
+		}
+	}
+
+	for key, value := range capitals {
+		capitals[key] = (value - min) / (max - min)
+	}
+
+	return capitals
```

Comment:
There might have an issue here. Say all other bikers are shit, then we expect them to all have a low social capital value. However say the SC is [0.1, 0.3, 0.4, 0.5], this will be scaled to -> [0.2, 0.6, 0.8, 1], which is not what we want.

Reply:
What I think we should do is to apply clipping of values instead

Reply:
Good spot! I think this all stems from the fact that this is missing the normalising step to bring it back to 1. So rather than clipping (I'm not entirely sure what our bounds would be for this), I suggest adding the code block
```go
//instantiate the sum variable
sumOfCapitals := 0.0

// inside the loop on line 50
sumOfCapitals += capitals[key]

//Line 51 onwards
for key, value := range capitals {
    capitals[key] = capitals[key]/sumOfCapitals
}
```

Reply:
I think that should fix it and comply with the scenario you mentioned Marco